civihr_default_theme/scss/components/_panel.scss

```diff
@@ -152,11 +152,13 @@ form {
     padding: 0;
     position: absolute;
     right: 20px;
-    top: 17px;
+    top: 13px;
```

Comment:
Please fix the order of the CSS rules. If this can be done in the whole file - that would be great. Please do so in a separate commit and do not mix with code changes.